Docker-compose.yml

```
@@ -0,0 +1,24 @@
+version: "3"
+
+services:
+  Development:
+    env_file:
+        - .env
+    build: . 
+    
+    .volumes: 
```

```suggestion
    volumes: 
```

Same for the one below.